Source/diablo.cpp

```diff
@@ -403,7 +404,7 @@ void ReleaseKey(int vkey)
 		CaptureScreen();
 	if (vkey == DVL_VK_MENU || vkey == DVL_VK_LMENU || vkey == DVL_VK_RMENU)
 		AltPressed(false);
-	if (vkey == DVL_VK_CONTROL || vkey == DVL_VK_LCONTROL || vkey == DVL_VK_RCONTROL)
+	if (vkey == DVL_VK_RCONTROL)

```

Comment:
Is this change needed?

Reply:
@AJenbo no, it's not needed although is very annoying dropping items while item highlight toggles.

Reply:
I will revert so someone else can take the decision of what to do about it in a future PR.

Reply:
Highlight should be moved to a keymapper option any way